Sources/SourceKitLSP/Swift/SwiftLanguageServer.swift

```diff
@@ -259,6 +262,20 @@ public final class SwiftLanguageServer: ToolchainLanguageServer {
       }
     })
   }
+  
+  /// Obtain the path of the directory to write generated module interfaces.
+  private static func generatedInterfacesPath() throws -> URL {
+    if let path = ProcessInfo.processInfo.environment["SOURCEKIT_GENERATED_INTERFACES_PATH"] {
```

Comment:
What do you think about making this an argument to the `sourcekit-lsp` executable instead of an opaque environment variable?

Reply:
I'll make that change.